website/templates/project/settings.mako

```diff
@@ -378,8 +378,31 @@
                     <div class="panel-heading clearfix">
                         <h3 class="panel-title">Email Notifications</h3>
                     </div>
+                    % if node['mailing_list_enabled'] or user['is_admin']:  ## Begin Project Mailing Lists Settings
```

Comment:
They weren't present several months ago when I first wrote this that part of the code. I'll do a bit of digging